libraries/chain/include/eosio/chain/apply_context.hpp

```diff
@@ -585,6 +585,9 @@ class apply_context {
       int32_t  kv_it_lower_bound(uint32_t itr, const char* key, uint32_t size, uint32_t* found_key_size, uint32_t* found_value_size);
       int32_t  kv_it_key(uint32_t itr, uint32_t offset, char* dest, uint32_t size, uint32_t& actual_size);
       int32_t  kv_it_value(uint32_t itr, uint32_t offset, char* dest, uint32_t size, uint32_t& actual_size);
+      kv_context& kv_get_backing_store() const {
+         return *kv_backing_store;
```

Comment:
Is this guaranteed to be non null?

Reply:
Thanks for spotting this! In theory kv_backing_store should have been created before kv_get_backing_store is called, but we cannot prevent it from being called prematurely. Will add an assert to check the pointer.